### PR TITLE
Create greetings.yml

### PR DESCRIPTION
**Pull request by MartinBraquet**
A new feature allowing to greet the new contributors (but also to check the compilation of the new documents if someone is interested in).

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Merci pour cette amélioration, ça fait toujours plaisir de voir de nouveaux contributeurs !'' first issue'
+        pr-message: 'Merci pour cette sugestion, ça fait toujours plaisir de voir de nouveaux contributeurs !'' first pr'
```
Comment (Peiffap) [inline .github/workflows/greetings.yml]:
```suggestion
        pr-message: 'Merci pour cette suggestion, ça fait toujours plaisir de voir de nouveaux contributeurs !'' first pr'
```

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Merci pour cette amélioration, ça fait toujours plaisir de voir de nouveaux contributeurs !'' first issue'
```
Comment (MartinBraquet) [inline .github/workflows/greetings.yml]:
```suggestion
        issue-message: 'Merci pour cette suggestion ça fait toujours plaisir de voir de nouveaux contributeurs !'' first issue'
```

File: .github/workflows/greetings.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        pr-message: 'Merci pour cette suggestion, ça fait toujours plaisir de voir de nouveaux contributeurs !'' first pr'
```
Comment (MartinBraquet) [inline .github/workflows/greetings.yml]:
```suggestion
        pr-message: 'Merci pour cette amélioration,, ça fait toujours plaisir de voir de nouveaux contributeurs !'' first pr'
```

**Comment by MartinBraquet** [comment]
> T'as inversé les messages de PR et d'issue non ? @MartinBraquet

Indeed

**Pull request merged**